colabs/wandb_registry/zoo_wandb.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<p>There is an oportunity to use wandb to track this split (using the function arguments as config), and then:</p><ul><li>Make a note that, although optional, tracking this split can simplify reproducibility later on</li><li>Make a note (with a doc link) explaining that doing this will also reflect the split in the lineage graph, which improves observability</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/560/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>